core/src/main/java/org/javarosa/xform/parse/XFormParser.java

```diff
@@ -324,20 +327,24 @@ private void initState() {
 
     public XFormParser(Reader reader) {
         _reader = reader;
+        extensionParsers = new Vector<QuestionExtensionParser>();

```

Comment:
These can be moved to the private Vector<> definition above for clarity/lack of repeating